src/commands/test.rs

```diff
@@ -0,0 +1,352 @@
+use crate::commands::utils::balance::{
+    display_balance, display_l1_balance, get_erc20_balance_decimals_symbol,
+};
+use crate::commands::utils::wallet::*;
+use crate::config::ZKSyncConfig;
+use clap::Subcommand;
+use colored::*;
+use eyre::ContextCompat;
+use spinoff::{spinners, Color, Spinner};
+use std::ops::Div;
+use zksync_ethers_rs::contracts::erc20::MINT_IERC20;
+use zksync_ethers_rs::{
+    core::{k256::ecdsa::SigningKey, rand::thread_rng, utils::parse_ether},
+    providers::{Http, Middleware, Provider},
+    signers::{LocalWallet, Signer, Wallet},
+    types::{
+        transaction::eip2718::TypedTransaction, Address, Eip1559TransactionRequest, L2TxOverrides,
+        TransactionReceipt, U256,
+    },
+    wait_for_finalize_withdrawal,
+    zk_wallet::ZKWallet,
+    ZKMiddleware,
+};
+
+#[derive(Subcommand, PartialEq)]
+pub(crate) enum Command {
+    #[clap(about = "LoadTest the zkStack Chain.")]
+    LoadTest {
+        #[clap(long = "wallets", short = 'w', required = true)]
+        number_of_wallets: u16,
+        #[clap(
+            long = "amount",
+            short = 'a',
+            required = true,
+            help = "Amount of BaseToken to deposit, 20% more will be deposited.\nThat extra 20% will remain in the main wallet,\nthe rest will be redistributed to random wallets"
+        )]
+        amount: f32,
+        #[clap(
+            long = "reruns",
+            short = 'r',
+            help = "If set to 0 it will run indefinitely, If not set defaults to 1 run."
+        )]
+        reruns_wanted: Option<u8>,
+    },
+    #[clap(about = "Mint ERC20 token on L1.")]
+    Erc20L1Mint {
+        #[clap(long = "token")]
+        token_address: Option<Address>,
+        #[clap(long = "amount", short = 'a', default_value = "10")]
+        amount: String,
+    },
+}
+
+impl Command {
+    pub async fn run(self, cfg: ZKSyncConfig) -> eyre::Result<()> {
+        let wallet_config = cfg.wallet.clone().context("Wallet config missing")?;
+        match self {
+            Command::LoadTest {
+                number_of_wallets,
+                amount,
+                reruns_wanted,
+            } => {
+                let l1_provider = Provider::try_from(
+                    cfg.network
+                        .l1_rpc_url
+                        .context("L1 RPC URL missing in config")?,
+                )?;
+                let l2_provider = Provider::try_from(cfg.network.l2_rpc_url)?;
+
+                let wallet = wallet_config.private_key.parse::<Wallet<SigningKey>>()?;
```

Comment:
Nit, you can parse a private key `String` to a `LocalWallet`.

Reply:
fixed